diff --git a/dist/module.mjs b/dist/module.mjs
index 8da96e2dd2fcc4c161ee24d63798302c4fd585e0..04da43f230f994db6a0d8a4b00363dc6d5f74583 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -3,7 +3,7 @@ import { extendViteConfig, useNitro, defineNuxtModule, createResolver, addServer
 import { defu } from 'defu';
 import { resolve } from 'pathe';
 import fs from 'node:fs/promises';
-import { bundledLanguagesInfo } from 'shiki/bundle/full';
+import { bundledLanguagesInfo } from 'shiki/langs';
 import { pascalCase } from 'scule';
 export { defineConfig } from './config.mjs';
 
